### PR TITLE
Add configuration file for orchestration

### LINKED ISSUE #56: Use configuration file for integration with other modules
Instead of using credcreator.py for storing configuration,

[./orchestration/connectionmanager/credcreator.py set "St2" "<ST2 Endpoint IP>" "st2admin" "<ST2 Password>"]

add a configuration file.

**Is this a BUG REPORT or FEATURE REQUEST?**:

 /kind feature

### PR DESCRIPTION
**Pull request by joseph-v**
**What this PR does / why we need it**:
This PR will add configuration file for Orchestration and unit tests. 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #56

### CODE REVIEW

File: orchestration/utils/config.py [hunk 1]
```diff
@@ -49,7 +53,68 @@ def init_logging():
...
+    tech = ''
+    server = ''
+    user = ''
+    passwd = ''
+
+    try:
+        server = conf.get('workflow', 'host')
+        user = conf.get('workflow', 'username')
+        tech = conf.get('workflow', 'tech')
+        encripted = conf.get('workflow', 'encripted_password')
```
Comment (kumarashit) [inline orchestration/utils/config.py]: encripted => encrypted

File: orchestration/utils/config.py [hunk 2]
```diff
@@ -49,7 +53,68 @@ def init_logging():
...
+        if encripted == 'false':
+            passwd = conf.get('workflow', 'password')
+        else:
+            phrase = conf.get('workflow', 'phrase')
+            ciphered_suite = Fernet(phrase.encode())
+            passwd = (ciphered_suite.decrypt(passwd.encode()))
+    except Exception as ex:
+        print(passwd)
```
Comment (kumarashit) [inline orchestration/utils/config.py]: remove this as if the passwd is not encrypted, we may print it on console. Also, can we use logger module to log this than to print on stdout

File: tests/unit/utils/orchestration_test.conf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+[orchestration]
+host = 127.0.0.1
+port = 5000
+
+[workflow]
+tech = St2
+host = 10.0.0.1
+username = st2admin
+encripted_password = false
```
Comment (kumarashit) [inline tests/unit/utils/orchestration_test.conf]: encripted => encrypted

File: orchestration/utils/config.py [hunk 4]
```diff
@@ -49,7 +53,68 @@ def init_logging():
     logger.addHandler(server_log_file)
 
 
+def init_config(file='orchestration.conf'):
+    global conf
+    global HOST
+    global PORT
+
+    conf = configparser.ConfigParser()
```
[the remark this answers is not in the document]
Comment (joseph-v) [reply orchestration/utils/config.py]: Done

File: orchestration/utils/config.py [hunk 5]
```diff
@@ -49,7 +53,68 @@ def init_logging():
...
+    dataset = conf.read(file)
+    if len(dataset) == 0:
+        logger.error("Failed to open orchestration config file: [%s]" % file)
+        file = 'orchestration.conf'
```
[the remark this answers is not in the document]
Comment (joseph-v) [reply orchestration/utils/config.py]: Refactored logic for config file

File: orchestration/utils/config.py [hunk 6]
```diff
@@ -49,7 +53,68 @@ def init_logging():
...
+        logger.warning("Creating default config file at: [%s]" % file)
+        # create default template config file
+        cfgfile = open(file, 'w')
```
[the remark this answers is not in the document]
Comment (joseph-v) [reply orchestration/utils/config.py]: Done

File: orchestration/utils/config.py [hunk 7]
```diff
@@ -49,7 +53,68 @@ def init_logging():
...
+        conf.add_section('orchestration')
+        conf.set('orchestration', 'host', HOST)
+        conf.set('orchestration', 'port', PORT)
+        conf.add_section('workflow')
+        conf.set('workflow', 'tech', 'St2')
+        conf.set('workflow', 'host', '127.0.0.1')
+        conf.set('workflow', 'username', 'St2')
+        conf.set('workflow', 'encripted_password', 'false')
+        conf.set('workflow', 'password', 'password')
+        conf.set('workflow', 'phrase', '')
```
[the remark this answers is not in the document]
Comment (joseph-v) [reply orchestration/utils/config.py]: We can modify default config file to add this in future

File: orchestration/utils/config.py (hunk 1 above)
Comment (joseph-v) [reply orchestration/utils/config.py]: Done

File: orchestration/utils/config.py (hunk 2 above)
Comment (joseph-v) [reply orchestration/utils/config.py]: Done

File: tests/unit/utils/orchestration_test.conf (hunk 3 above)
Comment (joseph-v) [reply tests/unit/utils/orchestration_test.conf]: Done, Thank you for review

File: orchestration/utils/config.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (kumarashit) [reply orchestration/utils/config.py]: > 
> 
> We can modify default config file to add this in future

Ok

**Pull request merged**